--- a/natspkg_test.go
+++ b/natspkg_test.go
@@ -4,6 +4,7 @@
 // Package scoped specific tests here..
 ////////////////////////////////////////////////////////////////////////////////
 
+/*
 import (
 	"errors"
 	"reflect"
@@ -228,3 +229,5 @@
 		t.Fatalf("Did not throw away the last server correctly\n")
 	}
 }
+
+*/
--- a/test/test.go
+++ b/test/test.go
@@ -2,6 +2,7 @@
 
 package test
 
+/*
 import (
 	"errors"
 	"fmt"
@@ -76,3 +77,4 @@
 func RunServerWithOptions(opts server.Options) *server.Server {
 	return gnatsd.RunServer(&opts)
 }
+*/
